src/python/yaramod_python.cpp

```diff
@@ -117,6 +117,7 @@ void addBasicClasses(py::module& module)
 		.def_property_readonly("rules", &YaraFile::getRules)
 		.def_property_readonly("imports", &YaraFile::getImports)
 		.def_property_readonly("text_formatted", [](const YaraFile& self) { return self.getTextFormatted(); })
+		.def_property_readonly("tokenstream", [](const YaraFile& self) { return *(self.getTokenStream());} )
```

Comment:
Does that mean that accessing `tokenstream` will make copy of the whole tokenstream or will it return by reference?

Reply:
This makes a copy of the `TokenStream` which is not the desired behaviour. Thank you for pointing it out!

Reply:
I fixed it in the last commit [a113ee3](https://github.com/avast/yaramod/pull/101/commits/a113ee3b69c14490b7d815f8902f8ff552826aff). I tested that it returns by reference now.